modules/ROOT/pages/whatsnew.adoc

```diff
@@ -25,6 +25,13 @@ This can only happen during a mixed mode of SGW with a 3.2 version and an older
 
 --
 
+* *Additional Rev Cache Configuration Options*
+
+You can now specify a memory limit on your rev cache with the `rev_cache.max_memory_count_mb` configuration option.
+You can use this option alongside existing configurable limits to the number of documents in the rev cache to reduce the risk of OOM issues.
+
+For more information, see xref:configuration-schema-database.adoc#cache-rev_cache-max_memory_count_mb[]
```

Comment:
@ElliotFrancisHunter this link does not present with a user friendly text, e.g. see rev cache memory limit config, but with an internal structure with the #. Can that be changed?

Reply:
It seems antora cannot automatically populate xrefs with user friendly text from the anchored header. Easy fix is to manually add the text in the square brackets myself, good spot!

Reply:
Change is in! Updating preview now